src/test/java/ru/training/at/hw2/BaseTest.java

```diff
@@ -0,0 +1,58 @@
+package ru.training.at.hw2;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.asserts.SoftAssert;
+
+import java.util.concurrent.TimeUnit;
+
+public class BaseTest {
+
+    public WebDriver webDriver = null;
+
+    @BeforeClass    //setting up the webdriver for Chrome
+    public void setupWebDriver() {
+        if (webDriver == null) {
+            WebDriverManager.chromedriver().setup();
+            webDriver = new ChromeDriver();
+//            webDriver.manage().window().maximize();
```

Comment:
Why this line commented

Reply:
I forgot to uncomment this line. I will fix that